src/binary/coreir.cpp

```diff
@@ -49,6 +49,9 @@ int main(int argc, char* argv[]) {
     "n,namespaces",
     "namespaces to output: '<namespace1>,<namespace2>,<namespace3>,...'",
     cxxopts::value<std::string>()->default_value("global"))(
+    "verilog-prefix",
```

Comment:
tbh I like dash instead of underscore for cmd line args, but seems like the standard is underscores

```suggestion
    "verilog_prefix",
```

Reply:
Standard as in general practice or standard is in coreir? I think using dash is standard in general practice, there are some coreir options that don't use it but I think those should be changed, rather than changing this.

From https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html
> Option names are typically one to three words long, with hyphens to separate words.

Reply:
agreed, dash is *standard* standard (and better), just meant as far as the CoreIR repo is concerned. We can keep this and change the others to dashes, agreed.